source/includes/kubernetes/_k8_daemonsets.md

```diff
@@ -76,3 +76,73 @@ Retrieve a daemon set and all its info in a given [environment](#administration-
 | `status`<br/>_object_                      | The status information of the daemon set                        |
 
 Note that the list is not complete, since it is refering to the [kubernetes api details](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md).
+
+<!-------------------- CREATE DAEMON SET -------------------->
+
+### Create a daemon set
+```shell
+curl -X POST \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/pods"
+  Content-Type: application/json
+  {
+  "apiVersion": "apps/v1",
+  "metadata": {
+    "name": "daemonset-name",
+    "namespace": "default"
+  },
+  "spec": {
+    "selector": {
+      "matchLabels": {
+        "name": "fluentd-elasticsearch"
+      }
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "name": "fluentd-elasticsearch"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "fluentd-elasticsearch",
+            "image": "quay.io/fluentd_elasticsearch/fluentd:v2.5.2"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "1542bd45-4732-419b-87b6-4ea6ec695c2b",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/daemonsets</code>
+
+Create a daemon set in a given [environment](#administration-environments).
+
+| Attributes                                 | &nbsp;                                                                    |
+| ------------------------------------------ | --------------------------------------------------------------------------|
+| `apiVersion` <br/> _string_                | The api version (versioned schema) of the daemon set                      |
```

Comment:
Wondering if we should specify which of the attributes are required 🤔 . Its hard because even the Kubernetes API isnt entirely clear what is required and what is not.

Reply:
I think for sure some of these we can enforce like `metadata` and `metadata.name`. All persisted objects in Kubernetes are required. 

Most also require the `namespace`. (All the ones you introduced do for sure) - there are some exceptions like cluster roles, and persistent volumes. 

We also know `spec.containers` for Pods is required as well

Reply:
Sure! How should I specify that they're required? Bolding? New column that says "required"?

Reply:
There are some examples in the docs if you search for "required". It's usually another table.

Reply:
It looks like we typically split it into two tables 
![Screen Shot 2020-07-10 at 9 40 17 AM](https://user-images.githubusercontent.com/11682191/87160750-74921a80-c291-11ea-9456-c5971d76d70d.png)
![Screen Shot 2020-07-10 at 9 39 47 AM](https://user-images.githubusercontent.com/11682191/87160751-752ab100-c291-11ea-9b91-342d66c97469.png)

Reply:
y'all are my heroes 💕

Reply:
Done! cc94a073d874ada178c6c92399ae0e3921f9dcb5